src/viam/resource/registry.py

```diff
@@ -152,28 +132,8 @@ def lookup_subtype(cls, subtype: "Subtype") -> ResourceRegistration:
                 raise ResourceNotFoundError(subtype.resource_type, subtype.resource_subtype)
 
     @classmethod
-    def lookup_component(cls, subtype: "Subtype", model: "Model") -> "ComponentCreator":
-        """Lookup and retrieve a registered component by its name
-
-        Args:
-            subtype (Subtype): The Subtype of the component
-            model (Model): The Model of the component
-
-        Raises:
-            ResourceNotFoundError: Raised if the Subtype Model pairing is not registered
-
-        Returns:
-            ComponentCreator: The function to create the component
-        """
-        with cls._lock:
-            try:
-                return cls._COMPONENTS[f"{subtype}/{model}"]
-            except KeyError:
-                raise ResourceNotFoundError(subtype.resource_type, subtype.resource_subtype)
-
-    @classmethod
-    def lookup_service(cls, subtype: "Subtype", model: "Model") -> "ServiceCreator":
-        """Lookup and retrieve a registered service by its name
+    def lookup_resource(cls, subtype: "Subtype", model: "Model") -> "ResourceCreator":
```

Comment:
nit: `lookup_creator` or `lookup_resource_creator`? `resource` is slightly ambiguous here

Reply:
What if we mirror the way we do subtypes? We have `register_subtype`, `lookup_subtype`, and `SUBTYPES`. We could do `register_resource_model`, `lookup_resource_model`, and `RESOURCE_MODELS`? Or `register_resource_creator`, ...

Reply:
I like `resource_creator` as less ambiguous of the two